src/Parser.ts

```diff
@@ -208,7 +214,7 @@ export class Parser {
   /**
    * Parse Inline Tokens
    */
-  parseInline(tokens, renderer) {
+  parseInline(tokens: Token[], renderer?: _Renderer | _TextRenderer): string | undefined {
```

Comment:
```suggestion
  parseInline(tokens: Token[], renderer?: _Renderer | _TextRenderer): string {
```

any undefined should be an empty string.